### PR TITLE
Crosswalk package content updates

### PR DESCRIPTION
**Pull request by gthuang**
Making some content updates to some of the Crosswalk registry pages.

### CODE REVIEW

File: themes/default/content/registry/packages/eks/_index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ meta_desc: Learn how you can use Pulumi's Amazon EKS component to set up an AWS
 layout: overview
 ---
 
-Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code.
+Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code. This component exposes the [Crosswalk for AWS](({{<relref "/docs/guides/crosswalk/aws">}})) functionality documented in the [Pulumi Amazon EKS guide]({{< relref "/docs/guides/crosswalk/aws/eks/" >}}).
```
Comment (gthuang) [inline themes/default/content/registry/packages/eks/_index.md:7]: @susanev the relref to the crosswalk docs page is not working here for some reason. do you know why?

File: themes/default/content/registry/packages/aws-apigateway/_index.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ meta_desc: Easily create AWS API Gateway REST APIs using Pulumi.
 layout: overview
 ---
 
-Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the Crosswalk for AWS functionality documented in the [Pulumi AWS API Gateway guide]({{< relref "/docs/guides/crosswalk/aws/api-gateway" >}}) as a package available in all Pulumi languages.
+Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](({{<relref "/docs/guides/crosswalk/aws">}})) functionality documented in the [Pulumi AWS API Gateway guide]({{< relref "/docs/guides/crosswalk/aws/api-gateway" >}}) as a package available in all Pulumi languages.
```
Comment (gthuang) [inline themes/default/content/registry/packages/aws-apigateway/_index.md:7]: @susanev the relref to the crosswalk docs page is not working here also

File: themes/default/content/registry/packages/eks/_index.md (hunk 1 above)
Comment (susanev) [reply themes/default/content/registry/packages/eks/_index.md:7]: i dont think we are supposed to be using relref anymore i would instead use the regular link. i don't think the actual link will work for you until this gets to docs though, unless you pull this package down locally.

`[Crosswalk for AWS](/docs/guides/crosswalk/aws/)`

File: themes/default/content/registry/packages/aws-apigateway/_index.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ meta_desc: Easily create AWS API Gateway REST APIs using Pulumi.
 layout: overview
 ---
 
-Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the Crosswalk for AWS functionality documented in the [Pulumi AWS API Gateway guide]({{< relref "/docs/guides/crosswalk/aws/api-gateway" >}}) as a package available in all Pulumi languages.
+Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](/docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](/docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```
Comment (susanev) [inline themes/default/content/registry/packages/aws-apigateway/_index.md:7]:
```suggestion
Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](../docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](../docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```

File: themes/default/content/registry/packages/aws-apigateway/_index.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ meta_desc: Easily create AWS API Gateway REST APIs using Pulumi.
 layout: overview
 ---
 
-Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the Crosswalk for AWS functionality documented in the [Pulumi AWS API Gateway guide]({{< relref "/docs/guides/crosswalk/aws/api-gateway" >}}) as a package available in all Pulumi languages.
+Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](../docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](../docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```
Comment (susanev) [inline themes/default/content/registry/packages/aws-apigateway/_index.md:7]:
```suggestion
Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](~/docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](~/docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```

File: themes/default/content/registry/packages/aws-apigateway/_index.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ meta_desc: Easily create AWS API Gateway REST APIs using Pulumi.
 layout: overview
 ---
 
-Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the Crosswalk for AWS functionality documented in the [Pulumi AWS API Gateway guide]({{< relref "/docs/guides/crosswalk/aws/api-gateway" >}}) as a package available in all Pulumi languages.
+Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](~/docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](~/docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```
Comment (susanev) [inline themes/default/content/registry/packages/aws-apigateway/_index.md:7]:
```suggestion
Easily create AWS API Gateway REST APIs using Pulumi. This component exposes the [Crosswalk for AWS](../../../docs/guides/crosswalk/aws/) functionality documented in the [Pulumi AWS API Gateway guide](../../../docs/guides/crosswalk/aws/api-gateway/) as a package available in all Pulumi languages.
```

File: themes/default/content/registry/packages/awsx/_index.md [hunk 6]
```diff
@@ -9,7 +9,7 @@ This AWSx documentation is generated using the v1.0.0-beta series of AWSx. If yo
 the provider then we suggest you continue to use our [@pulumi/awsx]({{<relref "/docs/reference/pkg/nodejs/pulumi/awsx">}}) NodeJS documentation.
 {{% /notes %}}
 
-The Amazon Web Services (AWS) Crosswalk (AWSx) provider for Pulumi can provision many of the cloud resources available in [AWS](https://aws.amazon.com/). It uses the AWS SDK to manage and provision resources.
+The Amazon Web Services (AWS) [Crosswalk]({{<relref "/docs/guides/crosswalk/aws">}}) (AWSx) library uses automatic well-architected best practices to make common infrastructure-as-code tasks in [AWS](https://aws.amazon.com/) easier and more secure. It uses the AWS SDK to manage and provision resources.
```
Comment (susanev) [inline themes/default/content/registry/packages/awsx/_index.md:12]:
```suggestion
The Amazon Web Services (AWS) [Crosswalk](../../../docs/guides/crosswalk/aws">}}) (AWSx) library uses automatic well-architected best practices to make common infrastructure-as-code tasks in [AWS](https://aws.amazon.com/) easier and more secure. It uses the AWS SDK to manage and provision resources.
```

File: themes/default/content/registry/packages/eks/_index.md (hunk 1 above)
Comment (susanev) [inline themes/default/content/registry/packages/eks/_index.md:7]:
```suggestion
Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code. This component exposes the [Crosswalk for AWS](../../../docs/guides/crosswalk/aws) functionality documented in the [Pulumi Amazon EKS guide](../../../docs/guides/crosswalk/aws/eks/).
```

File: themes/default/content/registry/packages/awsx/_index.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ This AWSx documentation is generated using the v1.0.0-beta series of AWSx. If yo
 the provider then we suggest you continue to use our [@pulumi/awsx]({{<relref "/docs/reference/pkg/nodejs/pulumi/awsx">}}) NodeJS documentation.
 {{% /notes %}}
 
-The Amazon Web Services (AWS) Crosswalk (AWSx) provider for Pulumi can provision many of the cloud resources available in [AWS](https://aws.amazon.com/). It uses the AWS SDK to manage and provision resources.
+The Amazon Web Services (AWS) [Crosswalk](../../../docs/guides/crosswalk/aws">}}) (AWSx) library uses automatic well-architected best practices to make common infrastructure-as-code tasks in [AWS](https://aws.amazon.com/) easier and more secure. It uses the AWS SDK to manage and provision resources.
```
Comment (susanev) [inline themes/default/content/registry/packages/awsx/_index.md:12]:
```suggestion
The Amazon Web Services (AWS) [Crosswalk](../../../docs/guides/crosswalk/aws/) (AWSx) library uses automatic well-architected best practices to make common infrastructure-as-code tasks in [AWS](https://aws.amazon.com/) easier and more secure. It uses the AWS SDK to manage and provision resources.
```

File: themes/default/content/registry/packages/eks/_index.md [hunk 8]
```diff
@@ -4,7 +4,7 @@ meta_desc: Learn how you can use Pulumi's Amazon EKS component to set up an AWS
 layout: overview
 ---
 
-Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code.
+Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code. This component exposes the [Crosswalk for AWS](../../../docs/guides/crosswalk/aws) functionality documented in the [Pulumi Amazon EKS guide](../../../docs/guides/crosswalk/aws/eks/).
```
Comment (susanev) [inline themes/default/content/registry/packages/eks/_index.md:7]:
```suggestion
Amazon EKS is a Pulumi Component that creates and manages the resources necessary to run an [EKS Kubernetes cluster](https://aws.amazon.com/eks/) in AWS. Use this component to quickly set up an EKS cluster in just a few lines of code. This component exposes the [Crosswalk for AWS](../../../docs/guides/crosswalk/aws/) functionality documented in the [Pulumi Amazon EKS guide](../../../docs/guides/crosswalk/aws/eks/).
```

**Comment by susanev** [comment]
@gthuang alright i fixed these up for you

**Review by susanev — APPROVED** [review]

**Comment by meagancojocar** [comment]
LGTM 👍🏼

**Pull request merged**